graphql/schemabuilder/types.go

```diff
@@ -44,6 +44,14 @@ var Paginated fieldFuncOptionFunc = func(m *method) {
 	m.Paginated = true
 }
 
+// Inexpensive is an option that can be passed to a FieldFunc to indicate that
+// the function is inexpensive to execute, so it does not need to be parallelized.
+// Note that field funcs that do not accept a `context.Context` argument
+// are always considered inexpensive.
+var Inexpensive fieldFuncOptionFunc = func(m *method) {
```

Comment:
Perhaps there's a clearer name for this?  I used "inexpensive" for consistency with the existing Thunder code, but I can imagine other options (maybe "force serial execution" to more clearly connect the option with the change in runtime behavior?)